### ISSUE TITLE
[todo] Add option to bulk remove tasks or display current list after successful removal

### ISSUE THREAD

**Issue by mirfilip** [alexanderepstein/Bash-Snippets#119]
**Issue Label:**
* [x] New feature
* [x] Enhancement

**Description:**
I often have many tasks added and want to bulk-remove them later. With the current state, after you remove the task which is in the middle of the list, the numbering is rearranged. So, to remove many tasks but to avoid mistakes, I often do:
```
todo -r 5
todo -g 
todo -r 8
todo -g
todo -2
...
```

There are two ways to make it better:
1. Support removing multiple tasks at once:
```
todo -r 5 8 2
```
2. Always display current state of the list *after* successful removal of task:
```
todo -g
01). test 1
02). test 2
todo -r 1
01). test 2
```

**Comment by alexanderepstein** [comment]
I added displaying the current list after removal of a task in 4599cc14098cb75cb5cc155f6f10428405551c54. Adding bulk removal will be a little but harder of a task. However i will look inti implementing it in the near future

**Comment by alexanderepstein** [comment]
Added bulk remove in 4bad5305344f3b29a7554d6f01ab3badea3225da should be coming with the new release :smile_cat:

**Comment by alexanderepstein** [comment]
Released with [Version 1.18.1](https://github.com/alexanderepstein/Bash-Snippets/releases/tag/v1.18.1)
